dsc/src/main.rs

```diff
@@ -594,7 +619,7 @@ async fn start_dsc(
         println!("start ds: {:?}", ds.port);
         let txc = tx.clone();
         let dsc = ds.clone();
-        let (action_tx, action_rx) = watch::channel(Action::Start);
```

Comment:
`Action::Start` initialized the channel before, does it now need to be sent manually? should it be sent on the next line?

Reply:
The downstairs start happens automatically in the `ds_start_monitor` function.  That the `Action::Start` was being pre-loaded in the channel before was not an actual requirement for starting, and was either a requirement of the `watch::channel` or a bug in my understanding of how to setup the channel.